### PR TITLE
Separate out modules, publish them, and prepare for folks using twoslash/gatsby/etc

### PR DESCRIPTION
**Pull request by orta**
Big PR

### CODE REVIEW

File: packages/tsconfig-reference/copy/ja/options/allowJs.md [hunk 1]
```diff
@@ -3,16 +3,16 @@ display: "Allow JS"
 oneline: "Let TS include .JS files in imports"
 ---
 
-`.ts`、`.tsx`ファイルだけでなく、JavaScriptファイルをプロジェクトへインポートできるようにします。例えば、次のJSファイルを:
+`.ts`、`.tsx`ファイルだけでなく、JavaScript ファイルをプロジェクトへインポートできるようにします。例えば、次の JS ファイルを:
```
Comment (orta) [inline packages/tsconfig-reference/copy/ja/options/allowJs.md:6]: I've asked in the Japanese localization channel about these changes

File: packages/typescript-vfs/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "typescript-vfs",
-  "version": "0.1.0",
+  "name": "@typescript/vfs",
+  "version": "1.0.0",
   "license": "MIT",
```
Comment (orta) [inline packages/typescript-vfs/package.json:4]: Woooo! https://www.npmjs.com/package/@typescript/twoslash

File: packages/ts-twoslasher/package.json [hunk 3]
```diff
@@ -1,18 +1,18 @@
 {
-  "name": "ts-twoslasher",
-  "version": "0.1.0",
+  "name": "@typescript/twoslash",
+  "version": "0.1.1",
```
Comment (orta) [inline packages/ts-twoslasher/package.json:3]: Woo! https://www.npmjs.com/package/@typescript/twoslash

**Comment by orta** [comment]
I think there's a sneaky upgrade to prettier two somehow in here - maybe I should just do a manual version update

File: packages/tsconfig-reference/copy/ja/options/allowJs.md (hunk 1 above)
Comment (Quramy) [reply packages/tsconfig-reference/copy/ja/options/allowJs.md:6]: No problem 😄  Thanks for installing Prettier.

File: packages/gatsby-remark-shiki-twoslash/package.json [hunk 4]
```diff
@@ -1,22 +1,30 @@
 {
-  "name": "gatsby-remark-shiki",
-  "version": "0.1.0",
+  "name": "gatsby-remark-shiki-twoslash",
+  "version": "0.5.7",
```
Comment (orta) [inline packages/gatsby-remark-shiki-twoslash/package.json:3]: Woo: https://www.npmjs.com/package/gatsby-remark-shiki-twoslash

**Pull request merged**